### PR TITLE
Re-do read me structure

### PR DESCRIPTION
**Pull request by dubeyShivank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
 
 [![Deploy to Netlify from fork](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/dolbyio-samples/comms-app-react-events)
 
-## Running locally
+## Getting Started
 
-Please see [the events package readme](./events/README.md) for information on how to run the app locally.
+The following steps will quickly get you started testing the Dolby.io Communications APIs capabilities.
```
Comment (tommydolby) [inline README.md:48]: Don't this need line IMO

File: README.md [hunk 2]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+* [Yarn v1.22.19](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+* [Node v18.0.0 or higher](https://nodejs.org/en/download)
+
+### Basic terminology
+
+As you browse through the source code and documents, you might come across some of these terms
+
+* **A host** is a participant with additional permissions to manage the conference and other participants.
+* **A viewer** is a participant who can only receive video and audio stream from the conference.
+* A **proxy-app-server** is an intermediary API server that communicates with the Dolby.io Communications Platform in order to provide functionality such as RTS/RTMP/HLS streaming or consuming web-hook data. You can see our sample implementation [here](./backend).
```
Comment (tommydolby) [inline README.md:64]: **proxy-app-server** - do we actually use this exact term (hyphens included) anywhere?

File: README.md [hunk 3]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+To set up your app for events, you will need to:
+
+1. Go to the _Dashboard_, and click `add new app` if you do not have an existing app. ![dashboard](events/documentation//dashboard-events.png)
+2. To enable events streaming, your app should be opted into the open beta program. You can find this at the bottom of the `Communications APIs` sidebar navigation when you click on your app.
+
+ ![dashboard](events/documentation/open-beta.png)
+
+## How to run the Events app
+
+Run the following steps after cloning the repository to run the application locally.
```
Comment (tommydolby) [inline README.md:83]:
```suggestion
To run the application locally, clone the repository and then perform the following steps.
```

File: README.md [hunk 4]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+
+The code in this repository is organised in the following way
+
+* The `/events/src/` directory contains all the front-end code for the events app. Within this directory
+  * `hooks/` contains wrapper functions around our SDK for re-usable functionality.
+  * `components/` contains UI components that encapsulate and provide functionality.
+  * `utils/` provides some generic helper functions.
+  * `context/` contains the React Context for the side drawer and the main component window.
+
+* The `backend/` contains the code for the proxy server.
```
Comment (tommydolby) [inline README.md:105]:
```suggestion
* The `/backend/` contains the code for the proxy server.
```

Also, it'd look neater if the folders didn't have the trailing slashes 😄

File: README.md [hunk 5]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+
+Open a terminal window in the root directory of your project folder. Install the project's dependencies using the following command.
+
+```bash
+yarn
+```
+
+### Repo structure
+
+The code in this repository is organised in the following way
```
Comment (tommydolby) [inline README.md:97]: Probably don't need this line

File: README.md [hunk 6]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+
+* The `/events/src/` directory contains all the front-end code for the events app. Within this directory
```
Comment (tommydolby) [inline README.md:99]:
```suggestion
* The `/events/src/` directory contains all the front-end code for the events app. Within this directory:
```

File: README.md [hunk 7]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+  * `hooks/` contains wrapper functions around our SDK for re-usable functionality.
+  * `components/` contains UI components that encapsulate and provide functionality.
+  * `utils/` provides some generic helper functions.
+  * `context/` contains the React Context for the side drawer and the main component window.
+
+* The `backend/` contains the code for the proxy server.
+
+This project is built with the [Comms UI Kit for react](https://github.com/dolbyio/comms-uikit-react) library for simplicity and re-use of standard Communications API based components.
```
Comment (tommydolby) [inline README.md:107]:
```suggestion
This project is built with the [Comms UI Kit for React](https://github.com/dolbyio/comms-uikit-react) library for simplicity and re-use of standard Communications API-based components.
```

File: README.md [hunk 8]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+
+### Set up `.env` file
+
+Create a new file called `.env` in the `events` folder, and copy the contents of `.env.example` into it.
```
Comment (tommydolby) [inline README.md:111]:
```suggestion
Create a new file called `.env` in the `events` folder and copy the contents of `.env.example` into it.
```

File: README.md [hunk 9]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+
+You will need to provide the values for `KEY` and `SECRET` from your dolby.io app. To obtain your key and secret from the Dolby.io dashboard,
```
Comment (tommydolby) [inline README.md:113]:
```suggestion
You will need to provide the values for `KEY` and `SECRET` from your Dolby.io app. To obtain your key and secret from the Dolby.io dashboard:
```

Just for consistency

File: README.md [hunk 10]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+
+1. Go to the _Dashboard_, and click on the `API Keys` next to your application.
```
Comment (tommydolby) [inline README.md:115]:
```suggestion
1. Go to the _Dashboard_ and click on the `API Keys` next to your application.
```

File: README.md [hunk 11]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+   ![dashboard](events/documentation/dashboard.png)
+2. On the next screen, copy the `App key` and `App secret` and paste them in your `.env` file against the marked variables.
+   ![token](events/documentation/client_access_token.png)
+
+If you want chat and the ability to promote a viewer to host, you will need to provide your `PUBNUB_PUBLISH_KEY`, `PUBNUB_SUBSCRIBE_KEY`, and `PUBNUB_SECRET_KEY` as well. To setup PubNub, please go through the [enabling chat guide in our customization guide](customization.md/#enabling-chat-and-promoting-a-viewer-to-co-host).
```
Comment (tommydolby) [inline README.md:120]:
```suggestion
To enable chat and the ability to promote a viewer to host, you will need to provide your `PUBNUB_PUBLISH_KEY`, `PUBNUB_SUBSCRIBE_KEY`, and `PUBNUB_SECRET_KEY` as well. To setup PubNub, please go through the [enabling chat guide in our customization guide](customization.md/#enabling-chat-and-promoting-a-viewer-to-co-host).
```

File: README.md [hunk 12]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+
+The secrets will not be exposed on the frontend.
+
+#### Start the app
+
+Execute the following command to run the application locally in a terminal window inside the `/events/` directory.
```
Comment (tommydolby) [inline README.md:126]:
```suggestion
#### Running the app

To run the application locally, execute the command below inside the `/events` directory.
```

Also, we should change all of the headings to either use gerund (-ing words) or not (but not both)

File: README.md [hunk 13]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+
+```bash
+yarn dev
+```
+
+#### Open the app in a browser
+
+After the appropriate message appears in the terminal window, open <http://localhost:3000> in the browser. The application will launch at this address and will look like the screenshot below.
```
Comment (tommydolby) [inline README.md:134]:
```suggestion
Once the dev server starts, open <http://localhost:3000> in the browser. The application will launch at this address and will look like the screenshot below.
```

"appropriate message" is a bit ambiguous 😅

**Review by tommydolby — APPROVED** [review]
Structure is fine. I just felt like leaving general comments about the README 😄

File: README.md [hunk 14]
```diff
@@ -43,18 +43,104 @@ Refer to [this guide](#how-to-get-a-dolbyio-account) on how to obtain your Dolby
...
+
+* [NPM v8.11 or higher](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+* [Yarn v1.22.19](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+* [Node v18.0.0 or higher](https://nodejs.org/en/download)
+
+### Basic terminology
+
+As you browse through the source code and documents, you might come across some of these terms
+
+* **A host** is a participant with additional permissions to manage the conference and other participants.
```
[the remark this answers is not in the document]
Comment (dubeyShivank) [reply README.md:64]: I mean you're right - I copied them from our solutions guide. Let me amend them

File: README.md (hunk 2 above)
Comment (dubeyShivank) [reply README.md:66]: Nope, I've unhyphenated it. We do refer to Proxy App Server a fair bit.

**Pull request merged**